site/components/src/main/java/com/visitscotland/brxm/factory/ArticleFactory.java

```diff
@@ -27,15 +30,17 @@ public class ArticleFactory {
     private final LinkService linkService;
     private final QuoteFactory quoteEmbedder;
     private final AnchorFormatter anchorFormatter;
+    private final CommonUtilsService commonUtils;
 
     public ArticleFactory(ImageFactory imageFactory,
                           QuoteFactory quoteEmbedder,
                           LinkService linkService,
-                          AnchorFormatter anchorFormatter) {
+                          AnchorFormatter anchorFormatter, CommonUtilsService commonUtils) {
```

Comment:
Optional small point regarding formatting, to be consistent I think this should be separated onto a new line.

```suggestion
                          AnchorFormatter anchorFormatter,
                          CommonUtilsService commonUtils) {
```

Small request which doesn't effect functionality, it can be done post-merge! 😄